src/v/rpc/BUILD

```diff
@@ -0,0 +1,9 @@
+py_binary(
```

Comment:
FYI this is using the built in rules not the ones in rules python

Reply:
Also buildifier is a nice tool that can auto correct stuff like that IIRC.

https://github.com/bazelbuild/buildtools/tree/master/buildifier

Reply:
>FYI this is using the built in rules not the ones in rules python

Ah, thanks. Fixed.

>Also buildifier is a nice tool that can auto correct stuff like that IIRC.

I've been using it for formatting. Added tools/format-bazel, but looks like some of these tools (format-cc, format-bazel, clang-tidy stuff) can be wrapped up later and managed by bazel itself like `bazel run //:clang-format`. not sure that's good practice or not?

Reply:
My opinion here is to have stuff managed via bazel so that everyone is one the same version and the version is tied to what is checked out in the repo (not vtools which is seperate), but can introduce nice wrappers so you just type `bazel fmt {cc,bzl,py}` etc. The wrappers can be done via https://github.com/bazelbuild/bazelisk?tab=readme-ov-file#toolsbazel

Reply:
Also tools like this should always be binary inputs to bazel (don't have bazel compile them, but download precompiled binaries) because bazel will for bad reasons recompile sometimes (when $PATH changes for example). For buildifier I've used https://github.com/keith/buildifier-prebuilt (in general keith has a bunch of goodies for bazel laying around).

Reply:
Alright I've put down the foundation for this stuff in the this branch. tools/bazel script, prebuilt buildifier and rule, etc... kudos Keith!